assets/javascripts/modules/checkout/checkout.js

```diff
@@ -1,12 +1,13 @@
 define(['$',
         'bean',
         'modules/checkout/form-elements',
-        'modules/checkout/validations'
-    ],
+        'modules/checkout/validations',
+        'utils/ajax'],
```

Comment:
Minor coding style point, probably move this up to after `bean` but before `modules/`